Lib/test/support/__init__.py

```diff
@@ -2039,22 +2039,23 @@ def threading_cleanup(*original_values):
     if not _thread:
         return
     _MAX_COUNT = 100
-    t0 = time.monotonic()
+
     for count in range(_MAX_COUNT):
         values = _thread._count(), threading._dangling
         if values == original_values:
             break
+
+        if not count:
+            environment_altered = True
+            print("Warning -- threading_cleanup() detected %s leaked threads "
```

Comment:
I don't like the "leak" terminology here. I think the previous message was better.

Reply:
Fixed. I reverted to the original message.